csharp/AppEncryption/AppEncryption/Persistence/IMetastore.cs

```diff
@@ -3,43 +3,39 @@
 
 namespace GoDaddy.Asherah.AppEncryption.Persistence
 {
+    /// <summary>
+    /// Primary interface for defining metastore operations.
+    /// </summary>
```

Comment:
Use the description text:

The Metastore interface provides methods that can be used to load and store system and intermediate keys from a
supported database.